File: Rally-FrontEnd/src/app/user-profile-arm/models/dto/UserBundleDTO.ts
import { RegisterDTO } from "./RegisterDTO";
import { UserInfoDTO } from "./UserInfoDTO";

export class UserBundleDTO {
    registerDTO: RegisterDTO;
    userInfoDTO: UserInfoDTO;

	constructor(registerDTO: RegisterDTO, userInfoDTO: UserInfoDTO){
        this.registerDTO = registerDTO,
        this.userInfoDTO = userInfoDTO
	}
}

File: Rally-BackEnd/backend/src/main/java/org/rally/backend/userprofilearm/model/dto/UserBundleDTO.java
package org.rally.backend.userprofilearm.model.dto;

public class UserBundleDTO {
    RegisterDTO registerDTO;
    UserInfoDTO userInfoDTO;

    public RegisterDTO getRegisterDTO() {
        return registerDTO;
    }

    public void setRegisterDTO(RegisterDTO registerDTO) {
        this.registerDTO = registerDTO;
    }

    public UserInfoDTO getUserInfoDTO() {
        return userInfoDTO;
    }

    public void setUserInfoDTO(UserInfoDTO userInfoDTO) {
        this.userInfoDTO = userInfoDTO;
    }
}
